Tekst o konferencjach branżowych - ze szczególnym uwzględnieniem konferencji o Pythonie 10 grudnia 2024 (#154)
techwriter-pl/techwriter-pl.github.io: blog/konferencje-online-patronujemy/index.md

> ---

Reviewer: Brakuje obrazka do postu. Myślę, że możesz użyć tego, który dostaliśmy z materiałami od organizatora: https://drive.google.com/file/d/1a73eiCa1A_aIUhB8S4t9qdur1SoWiMWq/view

title: 'Konferencje online - patronujemy'
authors: niedzwiedzka
date: '2024-12-05'
tags:
  - 'news'
  - 'z-kraju'
  - 'konferencje'
---
Pamiętacie, jak we wrześniu rekomendowaliśmy Wam serię darmowych konferencji online, połączonych z Wirtualnymi Targami Pracy? Cykl zbliża się właśnie do końca - czeka nas jeszcze jedna konferencja, ale za to jaka! 

<!--truncate-->
Przypomnijmy - do tej pory odbyły się:

- [Women TechStyle Summit 2024 (online) Autumn Edition](https://womentechstyle.pl/) - 19 września 2024 r., konferencja dla kobiet w IT
- [Javeloper 2024 (online) Autumn Edition](https://javeloper.pl/) - 26 września 2024 r., konferencja dla specjalistów Javy i architektury oprogramowania 
- [dev.js Summit 2024 (online) Autumn Edition](https://devjssummit.pl/) - 17 października 2024 r., konferencja o technologiach front-endowych i JavaScripcie
- [Expert Summit 2024 - All about Microsoft Technologies (online)](https://expertsummit.pl/) - 7 listopada 2024 r., konferencja o technologiach Microsoft

Byliśmy, sprawdziliśmy, polecamy!

> W kolejce czeka jeszcze:

Reviewer: ```suggestion
W kolejce czeka jeszcze [Pytech Summit 2024 (online) Winter Edition](https://pytechsummit.pl/). Ta konferencja dla pythonowców odbędzie się już za kilka dni, a dokładniej 10 grudnia 2024. 
```

> 

Reviewer: ```suggestion
```

> - [Pytech Summit 2024 (online) Winter Edition](https://pytechsummit.pl/) - 10 grudnia 2024 r. - konferencja dla pythonowców

Reviewer: ```suggestion
```

> Z naszego techwriterskiego punktu widzenia konferencja ta jest szczególnie cenna. Python to wszechstronny język programowania, używany m.in. do tworzenia stron z projektami dokumentacyjnymi, analizy danych i automatyzacji powtarzalnych, czasochłonnych zadań typu kopiuj - wklej. Pozwala też tworzyć wizualizacje i diagramy, ma dobre narzędzia do transformacji plików, np. pozwala szybko zrobić html z markdowna. Brzmi znajomo? Tak, społeczność techwriterska oraz programiści Pythona mogą podać sobie ręce. Co więcej, na konferencji dobrze odnajdą się także osoby z wykształceniem lingwistycznym - w programie znalazła się m.in. prezentacja o praktycznym wykorzystaniu dużych modeli językowych.

Reviewer: ```suggestion
Z naszego techwriterskiego punktu widzenia konferencja to wydarzenie jest szczególnie warte uwagi. Python to wszechstronny język programowania, używany m.in. do tworzenia stron z projektami dokumentacyjnymi, analizy danych i automatyzacji powtarzalnych, czasochłonnych zadań typu kopiuj-wklej. Pozwala też tworzyć wizualizacje i diagramy oraz ma dobre narzędzia do transformacji plików, np. z formatu HTML na Markdown. Brzmi znajomo? Tak, społeczność techwriterska oraz programiści Pythona mogą podać sobie ręce. Co więcej, na konferencji dobrze odnajdą się także osoby z wykształceniem lingwistycznym - w programie znalazła się m.in. prezentacja o praktycznym wykorzystaniu dużych modeli językowych.
```
